iosApp/iosAppTests/Support/TestContentController.swift

```diff
@@ -180,9 +190,14 @@ class TestContentController: MessengerHandler {
         }
     }
 
-    func waitForExpectation() {
-        let result = XCTWaiter().wait(for: [testExpectation!], timeout: 60)
-        XCTAssertEqual(result, .completed, "Expectation never fullfilled: \(testExpectation?.description ?? "No description.")")
+    func waitForTestExpectation(timeout: Double = 60.0) {
+        let result = XCTWaiter().wait(for: [testExpectation!], timeout: timeout)
+        XCTAssertEqual(result, .completed, "Test expectation never fullfilled: \(testExpectation?.description ?? "No description.")")
+    }
+    
+    func waitForErrorExpectatio(timeout: Double = 60.0) {
+        let result = XCTWaiter().wait(for: [errorExpectation!], timeout: timeout)
```

Comment:
You could set a guard before this line to make sure we have an errorExpectation available before running this.